force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +215,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result = '';
+        System.debug('inputstring = ' + inputString);
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        for (Integer i=0; i < repeatCount; i++){
+            result = result + inputString;
+            System.debug('i = ' + i);
+            if (i != repeatCount - 1) {
+                result = result + '; ';
+            }
```

Comment:
Good attention to detail ensuring that the last iteration does not have a semicolon